<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>591</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>SetVivadoPath</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>SetBoardPath</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="SetProjectDir">
       <property name="minimumSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>SetProjectDir</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTextBrowser" name="VivadoPath">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="SetPartName">
       <property name="minimumSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>9</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>SetPartName</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextBrowser" name="PartName">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextBrowser" name="BoardPath">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTextBrowser" name="ProjectDir">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>SetBoardName</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QTextBrowser" name="BoardName">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>15</width>
         <height>1</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="MakeProject">
       <property name="text">
        <string>MakeProject</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SetProjectDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetProjectDir_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetVivadoPath_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetBoardPath_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetBoardName_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetPartName</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetPartName_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MakeProject</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>MakeProject_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetProjectDir_Clicked()</slot>
  <slot>SetVivadoPath_Clicked()</slot>
  <slot>SetBoardPath_Clicked()</slot>
  <slot>SetBoardName_Clicked()</slot>
  <slot>SetPartName_Clicked()</slot>
  <slot>MakeProject_Clicked()</slot>
 </slots>
</ui>
